contents/docs/product-analytics/toolbar.mdx

```diff
@@ -8,38 +8,147 @@ availability:
-
-## How to activate the Toolbar
+## How to launch the toolbar
 
-1. Go to 'Launch Toolbar' in the left-hand menu
-2. Click 'Enable PostHog Toolbar'
-3. Add the URLs you wish to enable the toolbar for. Only you will be able to see it (and not your users).
+1. In your PostHog instance, Go to '[Launch Toolbar](https://app.posthog.com/toolbar)' in the left-hand menu
+2. Click 'Enable the PostHog toolbar'
+3. Add the URLs you wish to enable the toolbar for. Only you will be able to see it – not your users.
 4. Click Save and then Launch.    
 5. A new window will open with toolbar on your website. Click on the toolbar to interact with it.
 
 ![How to setup the PostHog toolbar](../../images/docs/product-analytics/toolbar-setup.mp4)
 
-## Toolbar features
-  
--   **Inspect.** With the PostHog Toolbar, you are able to get the `class` and `id` of an element much faster and without any extra noise. Click the element to inspect and you'll have access to everything you need to know about that element.
--   **Heatmap.** Easily see the ranking of which parts of the page your users are interacting with the most.
--   **Actions.** Inspect an element, see how often it has triggered an event, and immediately create an [Action](/docs/user-guides/actions) with that data. With the Toolbar, everything you need to tailor your analytics is right in front of you: your website, the user data, and the functionality to create an Action.
--   **Feature flags.** See which feature flags are turned on for a particular page.
+## Heatmaps 
+
+Heatmaps is a feature in the toolbar that shows you how users are interacting with elements on your website or app. 
+
+> **Note:** To use heatmaps, [autocapture](/docs/data/autocapture) must be enabled.
+
+To view your heatmaps, click on the toolbar and then click on the red fire (🔥) icon.
+
+![Launch Toolbar](../../images/tutorials/toolbar/toolbar-options.png)
+
+With the heatmap on, clickable elements on your website will have a red overlay with numbers on them. Here's an example of our website with the heatmap enabled:
+
+![An example from the PostHog website showing the heatmap highlighting a number of page elements. Some showing a count of clicks and one also showing a rage-click](../../images/tutorials/toolbar/posthog-heatmap-example.png)
+
+Each number in the top right box of each element represents the number of clicks it received. The number in the top left of the element is the number of rage-clicks. 
+
+Numbers at the bottom or top of the screen indicate that there are more elements that are not currently being displayed on the page.
+
+### How to create an action from a heatmap
+
+Once you have enabled the heatmap, click on any element. You will then be able to create an [action](/docs/data/actions) from it.
+
+![How to create an action using heatmaps](../../images/products/product-analytics/heatmaps-create-action.mp4)
```

Comment:
each of the mp4 links don't show in GitHub's markdown preview... is it just because GitHub doesn't know how to render them?

Reply:
yep, they work in the vercel preview ✅